### DISCUSSION TITLE
openapi/fets/server/integrations/aws-lambda

### DISCUSSION THREAD

**Discussion by giscus** [ardatan/feTS#420] [General]
# openapi/fets/server/integrations/aws-lambda

feTS Documentation

https://the-guild.dev/openapi/fets/server/integrations/aws-lambda

**Comment by hamedmam** [comment]
Hey all, thanks for making Spec first APIs with TS a real, I am really excited for this.

I am trying to get this work with Lambda currently and noticed in the example I get redirected to `/docs` without no content being shown, not sure what I miss but I can see lambda running locally on my machine and logs are coming through, it is just I can't seem to find where I can view the Swagger UI
